Vagrantfile

```diff
@@ -0,0 +1,32 @@
+# set up a Vagrantbox to run the repl
+# check it out at http://phpepl.dev after running 'vagrant up'
+
+Vagrant.configure("2") do |config|
+  config.vm.box = "ubuntu/trusty64"
+
+  # Remove the default Vagrant directory sync
+  config.vm.synced_folder ".", "/vagrant", disabled: true
+
+  config.vm.synced_folder "./", "/var/www/public", id: "vagrant-root",
+    owner: "vagrant",
+    group: "www-data",
+    mount_options: ["dmode=775,fmode=664"]
+
+  config.vm.network :private_network, ip: "192.168.56.101"
```

Comment:
I'm familiar with using vagrant, but not its set up. Is there any significance to this IP address? Would it ever need to be changed by a user?